### PR TITLE
Add docs for jwplayer and videojs

### PR DESCRIPTION
**Pull request by tsteur**

### CODE REVIEW

File: docs/media-analytics/custom-player.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In this guide we will learn how you can implement [Media Analytics](http://www.m
  
 ## Tracking a new media player in the browser
 
-The Media Analytics plugin currently supports  out of the box the HTML5, Vimeo and YouTube media players. If you are using
+The Media Analytics plugin currently supports out of the box the HTML5, Vimeo and YouTube media players. If you are using
```
Comment (mattab) [inline docs/media-analytics/custom-player.md]: Could we also here mention JW Player, VideoJS ? as people don't necessarily know that these players use html5

Comment (tsteur) [inline docs/media-analytics/custom-player.md]: I had it first here but then removed it again. That's why there is the space change. I'd personally rather not have it here. They are all based on HTML5 and we actually support many more players etc. We can add it, I just prefer not to

**Comment by mattab** [comment]
:+1: 

Will need to add it to User guide and FAQs too

**Comment by tsteur** [comment]
already done a 5 hours ago

**Pull request merged**